scriptmodules/emulators/pcsx2.sh

```diff
@@ -11,42 +11,56 @@
-        if [[ "$ini_value" != "0" ]]; then
-            if dialog --yesno "PCSX2 cannot be installed on a 64bit system with the RetroPie custom version of SDL2 installed due to version conflicts with the multiarch i386 version of SDL2.\n\nDo you want to downgrade to your OS version of SDL2 and continue to install PCSX2?" 22 76 2>&1 >/dev/tty; then
-                chown $user:$user "$configdir/all/retropie.cfg"
-                if rp_callModule sdl2 revert; then
-                    iniSet "own_sdl2" "0"
-                else
-                    md_ret_errors+=("Failed to install $md_desc")
-                fi
-            else
-                md_ret_errors+=("$md_desc install aborted.")
-            fi
-        fi
+        # We need to add the target architecture (no side effects if it's already added)
+        dpkg --add-architecture i386
+        # Installing compiler dependencies for crossbuild
+        getDepends gcc-multilib g++-multilib
     fi
+    # Build dependencies (from the Debian/Ubuntu package: https://github.com/PCSX2/pcsx2/blob/master/debian-packager/control)
+    getDepends cmake libaio-dev:i386 libasound2-dev:i386 libbz2-dev:i386 libgl1-mesa-dev:i386 libglu1-mesa-dev:i386 libgtk2.0-dev:i386 liblzma-dev:i386 libpng-dev:i386 libpulse-dev:i386 libpcap0.8-dev:i386 libsdl2-dev:i386 libsoundtouch-dev:i386 libwxbase3.0-dev:i386 libwxgtk3.0-dev:i386 libx11-dev:i386 libxml2-dev:i386 portaudio19-dev:i386 zlib1g-dev:i386
 }
 
-function install_bin_pcsx2() {
-    aptInstall pcsx2
+function sources_pcsx2() {
+    gitPullOrClone "$md_build" https://github.com/PCSX2/pcsx2.git master
 }
 
-function remove_pcsx2() {
-    aptRemove pcsx2
+function build_pcsx2() {
+    mkdir build
+    cd build
+    # Flags are the same as the Debian/Ubuntu package: https://github.com/PCSX2/pcsx2/blob/master/debian-packager/rules.
+    # More info at https://github.com/PCSX2/pcsx2/wiki/Installing-on-Linux.
+    # -DCMAKE_BUILD_TYPE=Release  -> Best in speed, but provides little or no debug/crash info 
+    # -DXDG_STD=TRUE              -> Use the Debian/Ubuntu configuration dir path, at ~/.config/PCSX2 
+    # -DPACKAGE_MODE=TRUE         -> Required to make it installable in different folders
+    # -DCMAKE_BUILD_STRIP=FALSE   -> Keep symbols. Better for debug. (recommended since it should not have any impact on speed)
+    # -DDISABLE_ADVANCE_SIMD=TRUE -> Disable AVX
+    # -DGSDX_LEGACY=TRUE          -> Build a GSdx legacy plugin compatible with GL3.3
+    cmake .. -DCMAKE_BUILD_TYPE=Release -DXDG_STD=TRUE -DPACKAGE_MODE=TRUE -DCMAKE_BUILD_STRIP=FALSE -DDISABLE_ADVANCE_SIMD=TRUE -DGSDX_LEGACY=TRUE -DCMAKE_TOOLCHAIN_FILE=cmake/linux-compiler-i386-multilib.cmake -DCMAKE_INSTALL_PREFIX="$md_inst"
+    make clean
+    make
+    md_ret_require="$md_build/build/pcsx2/PCSX2"
+}
+
+function install_pcsx2() {
+    cd build
+    make install
+    # Runtime dependencies
+    getDepends libasound2-plugins:i386 libspeexdsp1:i386 libusb-0.1-4:i386
```

Comment:
the install section can't use getDepends. These should be included in the main depends - but only the plugins - the dev packages include the libraries for the other ones

Reply:
I don't think libasound2-plugins is needed though actually.

Reply:
Moved dependencies to main depends. But I kept [the recommended packages](https://github.com/PCSX2/pcsx2/blob/master/debian-packager/control):
- libasound => alsa plugin for pulseaudio
- libusb    => ps3 controller (and probably others)